K8s-deployment/Charts/immudb/example-azure-resource-values.yaml

```diff
@@ -9,7 +9,7 @@ immudb:
       cpu: 1500m
       memory: 7Gi
   persistence:
-    storageClass: "azuredisk-csi-storage-class"
+    storageClass: "azuredisk-storage-class"
```

Comment:
Why is the storage class changed to non csi? Should use [`azuredisk-csi-storage-class`](https://github.com/datakaveri/iudx-deployment/blob/master/K8s-deployment/K8s-cluster/addons/storage/azure/azuredisk-csi-storage-class.yaml) itself.

Reply:
@pranavv0 please finish this